metabase/metabase
# fix stack display setting

Closes https://github.com/metabase/metabase/issues/39597
Closes https://github.com/metabase/metabase/issues/39598
Closes https://github.com/metabase/metabase/issues/39649

### Description

Fixes non-working stack display setting which should override series display settings:
<img width="356" alt="Screenshot 2024-03-13 at 9 22 52 PM" src="https://github.com/metabase/metabase/assets/14301985/942160f7-8b41-4e7c-96a1-d49ddc9d4cf5">

### How to verify

- Create a bar/area chart
- Change some series display settings to line
- Make it stacked
- Ensure the stack display setting overrides series display settings

### Demo

Before
<img width="1260" alt="Screenshot 2024-03-13 at 9 25 01 PM" src="https://github.com/metabase/metabase/assets/14301985/f22145ea-8a1c-40c5-ab8f-f1832ec14840">

After
<img width="1260" alt="Screenshot 2024-03-13 at 9 24 29 PM" src="https://github.com/metabase/metabase/assets/14301985/e5d00ed1-5a17-4e23-89f2-ef8860c08dc7">

### Checklist
